vcx/wrappers/node/src/utils/memory-management-helpers.ts

```diff
@@ -31,14 +24,7 @@ export abstract class GCWatcher {
     const release = this._releaseFn
     const handle = this._handleRef
     weak.addCallback(weakRef, () => {
-      try {
-        const rc = release(handle)
-        if (rc) {
-          throw rc
-        }
-      } catch (err) {
-        throw new VCXInternalError(err)
-      }
+      release(handle)
```

Comment:
This try/catch is no longer necessary due to the changes in the code above.

Reply:
Can you write a comment saying "why" we're not throwing an error? Unpredictable node GC for example.